main.go

```diff
@@ -122,7 +125,7 @@ func main() {
 	}
 
 	helm.ResourceDirectory = resourceDirectory
-	controller := controllers.NewIstioReconciler(mgr.GetClient(), mgr.GetScheme(), mgr.GetConfig(), resourceDirectory)
+	controller := controllers.NewIstioReconciler(mgr.GetClient(), mgr.GetScheme(), mgr.GetConfig(), resourceDirectory, strings.Split(defaultProfiles, ","))
```

Comment:
Does it play well if the user explicitly passes the empty string (i.e., they don't want any profiles to be applied automatically)? Do we have test for that?

Reply:
I've added a test and explicit handling of the empty value, even though it would be pretty hard to pass in an empty string to `--default-profiles`, since we set a default (you'd have to run the operator with `--default-profiiles=''` or `--default-profiles=`.